ifsbench/tests/test_pydantic_dataframe.py

```diff
@@ -72,3 +101,73 @@ def test_pydantic_data_frame_recover(default_frames):
 
     assert obj.frame_dict['name'].equals(copy.frame_dict['name'])
     assert obj.frame_list[0].equals(copy.frame_list[0])
+
+def test_pydantic_data_from_config_json(tmp_path, default_frames):
+    """
+    Check the from_config functionality, by 
+        * serialising an object with dump_config
+        * writing the resulting dict to JSON
+        * re-read from the JSON file
+        * call from_config to restore the original object
+        * check that the original and recovered object are equal.
+    """
+
+    obj = _PydanticDataFrameTest(
+        frame=default_frames[0],
+        frame_dict={'name': default_frames[1]},
+        frame_list=[default_frames[2]]
+    )
+
+    config = obj.dump_config()
+
+    path = tmp_path/'file.json'
+
+    with path.open('w') as f:
+        json.dump(config, f)
+    with path.open('r') as f:
+        data = json.load(f)
```

Comment:
This really just tests that json can read what it writes. You already tested above the the dumped config is sensible and easily human-readable, so I think you can skip this here.